package.json

```
@@ -4,7 +4,7 @@
   "license": "MIT",
   "scripts": {
     "ng": "ng",
-    "start": "ng serve",
+    "start": "ng serve --port 4100",
```

```suggestion
    "start": "ng serve",
```
Try to avoid committing changes to commands that may unintentionally affect development environments.